docassemble/MOHUDEvictionProject/data/questions/shared.yml

```diff
@@ -1295,7 +1295,7 @@ fields:
 
 ---
 code: |
-  if not users[0].attorney.there_are_any:
+  if users[0].attorney.target_number == 0:
```

Comment:
Maybe better: `if users[0].attorney.number_gathered() == 0`
but if this works, no problem

Reply:
Yes, I think this would be better, but I was afraid it wouldn't work for this tenant's saved interview - because of the problem with the review screen, there is an a users[0].attorney[0], but users[0].attorney.target_number was still set to 0. But maybe we should change it after we hear this tenant finishes her interview.

Reply:
Should I make an issue?